ruled_labels/[DRAFT]specs_monorepo.yaml

```diff
@@ -0,0 +1,160 @@
+    description: This is a runtime change that will require all nodes to be update BEFORE the runtime upgrade.
+    color: e8d7d7
+  - name: B3-add_new_host_function_to_client
+    description: PR adds a new host function to the client which is not yet used by the runtime.
+    color: f9f2f2
+  - name: B4-use_previously_unused_host_function_in_runtime
+    description: PR modifies the runtime to use a new host function; this raises the minimum client node version requirement for the next runtime upgrade!
+    color: f9f2f2
+  - name: B5-unused_host_function_breaking_change
+    description: PR introduces a breaking change to the behavior of a host function which is not used by any runtime.
+    color: f9f2f2
+  - name: B6-remove_unused_host_function
+    description: PR removes a host function not used by any runtime.
+    color: f9f2f2
+  - name: B7-breaking_change_RPC_API
+    description: This PR this changes the behavior of the external RPC API in a backwards incompatible way.
+    color: cdadad
+  - name: C0-easy
+    description: Can be fixed primarily by duplicating and adapting code by an intermediate coder.
+    color: eeeeee
+  - name: C1-medium
+    description: Can be fixed by a coder with good Rust knowledge but little knowledge of the codebase.
+    color: eebb00
+  - name: C2-substantial
+    description: Can be fixed by an experienced coder with a working knowledge of the codebase.
+    color: D4C5F9
+  - name: C3-involved
+    description: Can be fixed by an expert coder with good knowledge of the codebase.
+    color: E99695
+  - name: C4-mentor
+    description: An easy task where a mentor is available. Please indicate in the issue who the mentor could be.
+    color: "5319e7"
+  - name: I0-panic
+    description: The node panics and exits without proper error handling.
+    color: 17b25d
+  - name: I1-security
+    description: The node fails to follow expected, security-sensitive, behaviour.
+    color: d93f0b
+  - name: I2-bug
+    description: The node fails to follow expected behavior.
+    color: "249899"
+  - name: I3-annoyance
+    description: The node behaves within expectations, however this “expected behaviour” itself is at issue.
+    color: D6075D
+  - name: I4-refactor
+    description: Code needs refactoring.
+    color: 0075ca
+  - name: I5-enhancement
+    description: An additional feature request.
+    color: c2e0c6
+  - name: I6-meta
+    description: A specific issue for grouping tasks or bugs of a specific category.
+    color: a2eeef
+  - name: I7-duplicate
+    description: Issue is a duplicate. Closer should comment with a link to the duplicate.
+    color: D62209
+  - name: I8-wont_fix
+    description: Issue might be invalid or this project will not address it. Closer should comment why.
+    color: FB3701
+  - name: I9-optimisation
+    description: An enhancement to provide better overall performance in terms of time-to-completion for a task.
+    color: c5def5
+  - name: I10-unconfirmed
```

Comment:
I really like the Unconfirmed label that we currently have, and suggest keeping it.

This label will be auto-assigned to any issue that is opened by people outside of parity. Can this be done? 

This will then be used as the main guidance for those who regularly triage issues to quickly identify external issues (@juangirini, @bkchr, @ggwpez et al).

Reply:
we will keep the automation as is, just the code changed from `J2` to `I10`